packages/vuelidate/src/core.js

```diff
@@ -147,20 +148,19 @@ function createAsyncResult (rule, model, initResult, $pending) {
  * @param {String} key
  * @return {{$params: *, $message: Ref<String>, $pending: Ref<Boolean>, $invalid: Ref<Boolean>}}
  */
-function createValidatorResult (rule, state, key) {
-  const model = computed(() => state[key].value)
-  const ruleResult = callRule(rule.$validator, model)
+function createValidatorResult (rule, state, key, $dirty) {
+  const model = computed(() => unwrap(unwrap(state)[key]))
```

Comment:
whats with all the unwrapping?

Reply:
Oh damn. I think it might be unnecessary!

Reply:
Could you try to remove it and see if it works?

We will ofc cover this in tests later on.

Reply:
Unwrapped and it works nicely! Thanks!